fuzz/fuzz_targets/refcounted_model.rs

```diff
@@ -20,11 +17,19 @@ enum Operation {
 	Reference(u8),
 }
 
+#[derive(Clone, Debug)]
+struct Layer {
+	counts: [Option<i64>; 256],
+	is_maybe_saved: bool,
```

Comment:
`is_maybe_saved` is a bit confusing name for me, iiuc it indicates a layer has been written to disk, should we simply use `written`?